server/src/main/java/com/linecorp/centraldogma/server/command/StandaloneCommandExecutor.java

```diff
@@ -106,6 +108,10 @@ protected <T> CompletableFuture<T> doExecute(Command<T> command) throws Exceptio
             return (CompletableFuture<T>) createProject((CreateProjectCommand) command);
         }
 
+        if (command instanceof ResetMetaRepositoryCommand) {
```

Comment:
Note) I understood `ResetMetaRepositoryCommand` will be issued as soon as the metadata is migrated.
Otherwise if there are any updates by the users to the old metadata, a stale version may be read when `resetMetaRepository` is called.

Reply:
>  I understood ResetMetaRepositoryCommand will be issued as soon as the metadata is migrated

That is true. Also, I can check whether the repository is migrated correctly or not from this log after the command is executed:
https://github.com/line/centraldogma/pull/1123/files#diff-1b79753a6234a28cbdd41ef181ff57c2c3ed6d439b4e7fb6092994f35359d7beR214